src/vm/arm64/stubs.cpp

```diff
@@ -866,43 +843,47 @@ void HijackFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
 {
     LIMITED_METHOD_CONTRACT;
 
-    _ASSERTE(!"ARM64:NYI");
+     pRD->IsCallerContextValid = FALSE;
+     pRD->IsCallerSPValid      = FALSE;
+ 
+     pRD->pCurrentContext->Pc = m_ReturnAddress;
+     size_t s = sizeof(struct HijackArgs);
+     // stack must be multiple of 16. So if s is not multiple of 16 then there must be padding of 8 bytes 
+     s = s + (s%16 != 0) ? 8 : 0; 
```

Comment:
Shouldnt the padding account for the number of bytes required? What if S was multiple of 2 or 4 and not just 8? If it can only be multiple of 8, then should we assert that before adding the padding?